dist/api_data.py

```diff
@@ -820,8 +820,9 @@ def __cmp__(self, other):
 
 'WT_SESSION.drop' : Method([
     Config('checkpoint_wait', 'true', r'''
-        wait for the checkpoint lock, if \c checkpoint_wait=false, fail if
-        this lock is not available immediately''',
+        wait for the checkpoint lock, if \c checkpoint_wait=false, perform
+        the drop operation without taking a lock, failing in case a checkpoint
```

Comment:
Replace "failing in case a checkpoint is under progress" with "returning EBUSY if the drop operation conflicts with a running checkpoint". Similarly below.